### PR TITLE
#184749809 chore:setup git action

### PR DESCRIPTION
**Pull request by emilenfc**
# Description
add: setup of git action
## Type of change

Please check the options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature or Enhancement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have created a loom video for the new feature or enhancement
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .github/workflows/index.yml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build application
+        run: npm run build
```
Comment (SergeInGithub) [inline .github/workflows/index.yml:23]: We are using pnpm
```suggestion
        run: pnpm install

      - name: Build application
        run: pnpm run build
```

**Review by SergeInGithub — CHANGES_REQUESTED** [review]
Check the change i suggested

**Review by SergeInGithub — APPROVED** [review]
LGTM

**Review by shyakadavis — CHANGES_REQUESTED** [review]
Is there a specific reason why you are pushing the `build` folder, if not, you need to add `build` in the gitignore because it shouldn't be pushed.

The deployment service is the one which builds, and uses that folder.

I don't think we need it.

**Review by shyakadavis — APPROVED** [review]
LGMT 🚀

**Review by SergeInGithub — APPROVED** [review]
LGTM

**Review by mucyoalexaxel — CHANGES_REQUESTED** [review]

**Review by mucyoalexaxel — APPROVED** [review]
LGTM

**Pull request merged**